config/service_providers.localdev.yml

```diff
@@ -266,14 +266,14 @@ test:
     ial: 2
     help_text:
       sign_in:
-        en: <b>First time here from %{sp_name}?</b><p>Your old %{sp_name} username
+        en: <strong>First time here from %{sp_name}?</strong><p>Your old %{sp_name} username
```

Comment:
These probably didn't need to be updated. They represent an example of the help text a partner could upload to their app in the dashboard.

Certainly doesn't hurt, and will help avoiding getting caught in future searches! 😄